packages/valory/customs/kelly_criterion_no_conf/kelly_criterion_no_conf.py

```diff
@@ -53,6 +54,24 @@ def remove_irrelevant_fields(kwargs: Dict[str, Any]) -> Dict[str, Any]:
     return {key: value for key, value in kwargs.items() if key in ALL_FIELDS}
 
 
+def get_adjusted_kelly_amount(
+    info: Optional[str],
+    kelly_bet_amount: float,
+    weighted_accuracy: Optional[float],
+    static_kelly_fraction: float,
+):
+    """This function adjusts the kelly bet amount based on the weighted accuracy metric
+    of the selected tool to make the prediction. Default use-case: it uses the static kelly fraction
+    """
+    if weighted_accuracy is None:
+        info.append(f"No weighted accuracy information for this tool")
+        return int(kelly_bet_amount * static_kelly_fraction)
+    # weighted_accuracy must be always between [0, 1]
+    info.append(f"Weighted accuracy information: {weighted_accuracy}")
```

Comment:
Why is this comment here? Is some check missing?

Reply:
just adding a log about this parameter but I agree it could be placed somewhere else so removed

Reply:
I was asking about the comment and whether a check is missing.

Reply:
adding a check about the range